.gitignore

```diff
@@ -15,3 +15,5 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+*.swp
+.ruby-version
```

Comment:
.ruby-version is ignored, but it's added to the repo?

Reply:
lol good catch will remove